lambdas/common/src/database/exceptions/errors.ts

```diff
@@ -7,25 +9,41 @@ export class RecordExpiredError extends Error {
     super();
   }
 
-  get message() {
-    return `Found only expired records on the ${
-      this.tableName
-    } table for sessionId ${this.sessionId}: ${this.expiryDates
+  public get message() {
+    return `Found only expired records on the ${this.tableName} table: ${this.expiryDates
```

Comment:
Have we lost the closing }` here or is github being weird?

Reply:
Ah, the comment sort of obscures it - there's a .map that continues the expression underneath it:

![image](https://github.com/user-attachments/assets/15811f73-e7c3-4b7b-9742-9a13386f90be)

Reply:
ooooh maybe we extract the map into a var to put into the string?

Reply:
Works for me